site/topics/references/references.rst

```diff
@@ -96,63 +96,62 @@ Lists in Memory
 
-    Take 1 min and look at this picture and see if you can explain why we start counting at 0 when indexing lists/arrays.
+    Take a moment to look at this image and see if you can explain why we start counting at 0 when indexing lists.
    
 
-* Earlier we saw that lists work a little differently when saying something like
+* If we wanted to make a copy of ``z`` like we did with the integers ``x`` and ``y``\, we could write something like ``w = z``
+* And just like with the integers, this copies the contents of the memory location of ``z`` and puts it into a new location labelled with a ``w``
+* However, the catch is that the contents of ``z`` is the memory address of the list
+* After making the copy into ``w``,  how many references do have that get me to the memory location of the list?
 
->>> my_list = [1,2,3]
->>> another_list = my_list
->>> another_list[1] = 99
->>> print(my_list)
-[1, 99, 3]
- 
-* We called this aliasing and took note that it's weird
-* However... actually... the way we copy over ``my_list`` to ``another_list`` works THE SAME WAY AS PRIMITIVE TYPES
-    * But... You just said.. and you clearly showed us that it's totally different!!!!!!!!
+.. image:: array_pointer_copy.png
 
-* Strap yourselves in, because I'm about to blow your mind
+* If I want to make a change to ``w`` and I write something like ``w[4] = P``, the computer goes to the list referenced by ``w`` and alters the value at index ``4``
 
-* Let's say I write
+.. image:: array_pointer_copy_change.png
 
->>> w = z
+* This does not alter the contents of the memory location of ``w``\; this alters the data at the memory location that is referenced to by ``w``
+* In fact, this also alters the data at the memory location that is referenced by ``z``
 
-.. image:: array_pointer_copy.png
+    * What would happen if I wrote ``print(z[4])``\?
 
+* When we have two or more references referring to the same *thing* in memory, we call these aliases
 
-* Just follow the rules we followed for primitive types
-   * Copy over the contents of z to an open memory location
-   * Give it the label ``w``
-   
-* How many references do I now have that get me to the same memory location?
+    * ``w`` is an alias for ``z``
+    * This is discussed further in the following topic
 
-* Now let's look at what happens if I do this
 
->>> w[4] = P
+.. note::
 
-.. image:: array_pointer_copy_change.png
+    As mentioned earlier, in reality Python would not actually store the contents of the list within the block of memory
+    allocated for the list. Instead, Python actually stores the contents of the list in their own memory locations and
+    the memory location for the list stores references to the location of the respective contents. This detail is
+    important, but fortunately we could ignore this detail here for the purpose of learning about references. Further,
+    Python takes care of these details behind the scenes so we do not need to think about this too much.
```

Comment:
This is a bit wordy. Saying more simply that a list is itself also a reference to that array (among other details, like the length) would be sufficient here. References were just explained, I think it's fair enough to fall back on that, especially for something that is more of an in-passing note.